aspnetcore/performance/caching/memory.md

```diff
@@ -86,6 +96,32 @@ The following sample:
 
 [!code-csharp[](memory/sample/WebCache/Controllers/HomeController.cs?name=snippet_et&highlight=14-21)]
 
+::: moniker range=">= aspnetcore-2.0"
+
+## Use SetSize, Size, and SizeLimit to limit cache size
+
+The following code creates a unitless fixed size [MemoryCache](/dotnet/api/microsoft.extensions.caching.memory.memorycache?view=aspnetcore-2.1):
+
+[!code-csharp[](memory/sample/RPcache/Services/MyMemoryCache.cs?name=snippet)]
+
+[SizeLimit](/dotnet/api/microsoft.extensions.caching.memory.memorycacheoptions.sizelimit?view=aspnetcore-2.1#Microsoft_Extensions_Caching_Memory_MemoryCacheOptions_SizeLimit) does not have units. Cached entries must specify size. An entry will not be cached if the sum of the cached entry sizes exceeds the value specified by `SizeLimit`.
```

Comment:
Cached entries only have to specify size if the memory cache has a size limit set. If no size limit was set, the cache size set on the entry will be ignored.

Reply:
Cached entries must specify a size in whatever units they deem most appropriate. All users of a cache instance should use the same unit system.